GameServerLib/GameObjects/Spells/Spell.cs

```diff
@@ -95,7 +96,8 @@ public virtual bool Cast(float x, float y, float x2, float y2, IAttackableUnit u
             X2 = x2;
             Y2 = y2;
             Target = u;
-            FutureProjNetId = _networkIdManager.GetNewNetId();
+            uint FutureProjNetId = _networkIdManager.GetNewNetId();
```

Comment:
Not used?

Reply:
Do you mean take it out of the interface? It is used only in Spell.cs for NotifyCastSpell, and all the AddProjectile functions.

Reply:
This is local variable

Reply:
Yes, what about it?

Reply:
Where do you use it?

Reply:
Ok, I missed the usage.

Reply:
Do you really not need it? (to insert it into the projectiles list)

Reply:
If you want, I could delete the variable and put `_networkIdManager.GetNewNetId()` into `NotifyCastSpell`.

Reply:
From what I remember, you need to add this id to the first projectile (like Nidalee spear)

Reply:
Made it private and re-assigned netId after each projectile is added.